client/src/main/java/org/schlunzis/kurtama/client/settings/Setting.java

```diff
@@ -10,6 +12,8 @@ public class Setting<T> {
     public static final Setting<String> HOST = new Setting<>("host", "127.0.0.1");
     public static final Setting<Integer> PORT = new Setting<>("port", 8007);
     public static final Setting<String> LANGUAGE = new Setting<>("language", Locale.US.toLanguageTag());
+    public static final Setting<String> SERVER_PATH = new Setting<>("server-path", System.getProperty("user.home") + File.separator + ".kurtama");
```

Comment:
In linux this is a common way. And tbh. that's all I care for. But out of curiosity. Does windows follow a similar way of doing this? And what exactly is the home directory on windows?

Reply:
There are some folders like this one in my home directory, which is essentially the same as in linux. Something like C:\Users\<username>
There is something called AppData which might be a canonically better option on Windows. I have to look into this.